src/command.rs

```diff
@@ -24,6 +24,8 @@ pub enum Command {
     NextLayout,
     PreviousLayout,
     SetLayout,
+    FlipTagHorizontal,
```

Comment:
If we're gonna use shorthand (FlipTagVert), we might want to do so consistently.

Reply:
I'll change horizontal to horz. Should I update all of the horizontals and verticals to their short hands (most the verticals are in the layout names)

Reply:
Hmm, I don't see much other shorthand in the code to inform an opinion, and don't have an existing opinion on it myself. Sounds like a lot of work for not too much gain.

Reply:
I believe I remember stumbling over this inconsistency once in another place and was thinking I need to check later which one would be the more common used, but I don't remember where that was..